ocs_ci/helpers/helpers.py

```diff
@@ -4680,3 +4680,75 @@ def flatten_multilevel_dict(d):
         else:
             leaves_list.append(value)
     return leaves_list
+
+
+def verify_cephblockpool_status(
```

Comment:
It would be good to have functions from helpers.py in more specific modules related to their usage so that they can be easier found. E.g. `ocs_ci/ocs/rados_utils.py` for rados functionality.

Reply:
Addressed in latest commit